pkg/asset/manifests/openshift.go

```diff
@@ -73,6 +78,25 @@ func (o *Openshift) Generate(dependencies asset.Parents) error {
 				Base64encodeSecretAccessKey: base64.StdEncoding.EncodeToString([]byte(creds.SecretAccessKey)),
 			},
 		}
+
+	case azuretypes.Name:
+		resourceGroupName := clusterID.InfraID + "-rg"
+		session, err := azure.GetSession()
+		creds := session.Credentials
```

Comment:
should be using the session only **after checking for error**. this will panic when err not nil.

Reply:
good catch thanks

Reply:
fixed in latest